src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -14,7 +14,6 @@
 using Microsoft.ML.CommandLine;
 using Microsoft.ML.Data;
 using Microsoft.ML.EntryPoints;
-using Microsoft.ML.Internal.Internallearn;
```

Comment:
Are you merged with master? I thought that InternalLearn was removed in a prior change.

Reply:
Yep, maybe rebase did something.

---
In reply to: [261456781](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261456781) [](ancestors = 261456781)